diff --git a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
index 52450f0..1daa245 100644
--- a/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
+++ b/node_modules/react-native-safe-area-context/android/src/main/jni/CMakeLists.txt
@@ -36,6 +36,8 @@ if (REACTNATIVE_MERGED_SO)
           fbjni
           jsi
           reactnative
+          c++
+          c++abi
   )
 else()
   target_link_libraries(
@@ -56,6 +58,8 @@ else()
           rrc_view
           turbomodulejsijni
           yoga
+          c++
+          c++abi
   )
 endif()
 
