diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
index 7ce6058..476aa5e 100644
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -89,7 +89,7 @@ target_include_directories(
 set_target_properties(worklets PROPERTIES LINKER_LANGUAGE CXX)
 
 target_link_libraries(worklets log ReactAndroid::reactnative ReactAndroid::jsi
-                      fbjni::fbjni)
+                      fbjni::fbjni c++ c++abi)
 
 if(${JS_RUNTIME} STREQUAL "hermes")
   if(ReactAndroid_VERSION_MINOR GREATER_EQUAL 82)
